testdata/repos/gentoo/UseFlagsWithoutEffectsCheck/UseFlagWithoutDeps/UseFlagWithoutDeps-2.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Ebuild with unnecessary USE flags"
+HOMEPAGE="https://github.com/pkgcore/pkgcheck"
+
+SLOT="0"
+LICENSE="BSD"
+IUSE="bash-completion ipv6"
+
+src_compile() {
+	emake IPV6=$(use ipv6) BASH="$(use bash-completion)"
```

Comment:
Can we try add a test with 'usex' or something too? or 'use_with'?

Reply:
Oh, sorry, I forgot, this is not the same as the other check I was thinking of. Ignore!